pkgs/development/libraries/qt-6/modules/qtbase.nix

```diff
@@ -119,14 +121,16 @@ stdenv.mkDerivation rec {
     libpng
     pcre2
     pcre
-    libproxy
     zstd
-    double-conversion
     libb2
     md4c
+  ] ++ lib.optionals (!stdenv.hostPlatform.isMinGW) [
+    double-conversion
```

Comment:
`double-conversion` is now supported on windows since https://github.com/NixOS/nixpkgs/pull/290089

Reply:
fixed